enterprise/multi-node-tinkerbench/aerospike.conf

```
@@ -0,0 +1,62 @@
+# Aerospike database configuration file
+# This template sets up a single-node, single namespace developer environment.
+#
+# Alternatively, you can pass in your own configuration file.
+# You can see more examples at
+# https://github.com/aerospike/aerospike-server/tree/master/as/etc
+
+# This stanza must come first.
+service {
+	feature-key-file /etc/aerospike/features.conf
```

Align indentation with the rest of the file (spaces)
```suggestion
        feature-key-file /etc/aerospike/features.conf
```